### PR TITLE
Update configuration-vocabulary.md and trace-sampling.md

### PR DESCRIPTION
**Pull request by zifeihan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
Update configuration-vocabulary.md and trace-sampling.md

### CODE REVIEW

File: docs/en/setup/backend/configuration-vocabulary.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | agent-analyzer | default | Agent Analyzer. | SW_AGENT_ANALYZER | default |
 | - | -| sampleRate|Sampling rate for receiving trace. The precision is 1/10000. 10000 means 100% sample in default.|SW_TRACE_SAMPLE_RATE|10000|
 | - | - |slowDBAccessThreshold|The slow database access thresholds. Unit ms.|SW_SLOW_DB_THRESHOLD|default:200,mongodb:100|
+| - | - |forceSampleErrorSegment|When sampling mechanism active, this config can open(true) force save some error segment. true is default.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
```
Comment (wu-sheng) [inline docs/en/setup/backend/configuration-vocabulary.md:136]:
```suggestion
| - | - |forceSampleErrorSegment|When sampling mechanism activated, this config would  make the error status segment sampled, ignoring the sampling rate.|SW_FORCE_SAMPLE_ERROR_SEGMENT|true|
```

Comment (wu-sheng) [reply docs/en/setup/backend/configuration-vocabulary.md:136]: And change the sampling doc when same way. The previous version has some grammar issue.

File: docs/en/setup/backend/trace-sampling.md [hunk 2]
```diff
@@ -15,10 +15,18 @@ agent-analyzer:
   default:
     ...
     sampleRate: ${SW_TRACE_SAMPLE_RATE:1000} # The sample rate precision is 1/10000. 10000 means 100% sample in default.
+    slowDBAccessThreshold: ${SW_SLOW_DB_THRESHOLD:default:200,mongodb:100} # The slow database access thresholds. Unit ms.
```
Comment (wu-sheng) [inline docs/en/setup/backend/trace-sampling.md:18]: Why is this added in the sampling doc?

File: docs/en/setup/backend/trace-sampling.md [hunk 3]
```diff
@@ -15,10 +15,18 @@ agent-analyzer:
...
+    forceSampleErrorSegment: ${SW_FORCE_SAMPLE_ERROR_SEGMENT:true} # When sampling mechanism active, this config can open(true) force save some error segment. true is default.
 ```
 
-`sampleRate` is for you to set sample rate to this backend. 
-The sample rate precision is 1/10000. 10000 means 100% sample in default. 
+`sampleRate` is for you to set sample rate to this backend.
+The sample rate precision is 1/10000. 10000 means 100% sample in default.
+
+`slowDBAccessThreshold` is for you to set slow database access thresholds.
```
Comment (wu-sheng) [inline docs/en/setup/backend/trace-sampling.md:25]: Same question here.

Comment (wu-sheng) [reply docs/en/setup/backend/trace-sampling.md:25]: We don't sample the segment if the statement is slow, right?

Comment (zifeihan) [reply docs/en/setup/backend/trace-sampling.md]: @wu-sheng yes, We don't sampled segment if the statement is slow.

**Review by wu-sheng — APPROVED** [review]
